<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <arg name="rviz" default="true" />
  <param name="intrinsic_path"        value="$(find lidar_camera_fusion)/../../data/parameters/intrinsic.txt" />  
  <param name="extrinsic_path"        value="$(find lidar_camera_fusion)/../../data/parameters/extrinsic.txt" />  


  <node pkg="lidar_camera_fusion" type="lidar_camera_fusion" name="fusion_node" output="screen"/>

  <node pkg="tf" type="static_transform_publisher" name="livox_frame_pub" args="0 0 0 0 0 0 1 map livox_frame 10"/>


  <group if="$(arg rviz)">
    <node name="rviz" pkg="rviz" type="rviz" respawn="false" args="-d $(find lidar_camera_fusion)/launch/fusion.rviz" />
  </group>
</launch>

